src/edu/ithaca/dragonlab/ckc/ConceptKnowledgeCalculatorAPI.java

```diff
@@ -71,6 +71,7 @@
 
     void removeLORFile(String assessmentFile) throws Exception;
 
+    public String csvToResource() throws Exception;
```

Comment:
anything in the API needs good clear javadoc comments, but this especially because it returns a string explaining things, but it also has side effects of writing files.